scripts/lib-franklin.js

```diff
@@ -178,7 +179,7 @@ export async function decorateIcons(element) {
     const iconName = Array.from(span.classList).find((c) => c.startsWith('icon-')).split('-')[1];
     const parent = span.firstElementChild?.tagName === 'A' ? span.firstElementChild : span;
     // Styled icons need to be inlined as-is, while unstyled ones can leverage the sprite
-    if (ICONS_CACHE[iconName].styled) {
+    if (ICONS_CACHE[iconName]?.styled) {
```

Comment:
let's avoid optional chaining because of low browser support: https://caniuse.com/mdn-javascript_operators_optional_chaining

Reply:
The boilerplate uses optional chaining so I assumed it was fine to use it https://github.com/adobe/helix-project-boilerplate/blob/3823c125b4b50fc60a27f70f6dd79327dc6bd8ca/scripts/lib-franklin.js#L184 ?

Reply:
Anyways I remove it 👍